src/utils/s3_mods.py

```diff
@@ -0,0 +1,148 @@
+# General functions
+import json
+import pandas as pd
+from io import StringIO
+import logging
+from typing import List
+import subprocess
+import os
+
+# Functions specific to s3 bucket
+import boto3
+import raz_client
+from rdsa_utils.cdp.helpers.s3_utils import file_exists, create_folder_on_s3
```

Comment:
Import should be reordered in line with PEP8

1. Standard Library
2. Third Party
3. Local Imports

Reply:
Re-arranged